### ISSUE TITLE
A few suggestions/bugs i have

### ISSUE THREAD

**Issue by izmystic** [BerkieBb/bMenu#13]
- Misc > Menu Position doesnt seem to do anything (it only applies to new menus, such as the recording one)
- Misc > Display Location gives an error (@bMenu/client/menus/misc/main.lua:173: attempt to perform unsupported operation on a vector value (local 'currentPos'))
- Vehicle > Personal Vehicle Options > Set Vehicle creates a second Current Vehicle option

**Comment by BerkieBb** [comment]
> Misc > Menu Position doesnt seem to do anything (it only applies to new menus, such as the recording one)
This is because ox_lib only applies the menu position when a menu is reopened.

**Comment by izmystic** [comment]
> > Misc > Menu Position doesnt seem to do anything (it only applies to new menus, such as the recording one)
> 
> This is because ox_lib only applies the menu position when a menu is reopened.

even when reopening the main menu it stays in top-right

**Comment by BerkieBb** [comment]
Oh then it's probably because of how the menus are registered, the ones that it doesn't apply to are the ones that are static registered

**Comment by BerkieBb** [comment]
But with the permissions system most (or all) menus should be registered newly after everytime it should open

**Comment by BerkieBb** [comment]
Probably a case of the MenuOpen variable not resetting properly or something with ox_lib

**Comment by izmystic** [comment]
like the weather & time breaks as well when this happens

**Comment by BerkieBb** [comment]
Oh, so the whole code basically breaks then

**Comment by izmystic** [comment]
yeah thats what it seems like

**Comment by izmystic** [comment]
Killing other players doesnt work either, but it works on yourself

**Comment by BerkieBb** [comment]
Might be because of it being client side, this is where vMenu is outdated

**Comment by izmystic** [comment]
Noclip speed doesnt live update, but will show correct speed when turned on

**Comment by BerkieBb** [comment]
That worked last time I tried it, or I don't remember it lol

**Comment by izmystic** [comment]
let me rephrase, the text that says what speed doesnt update until turning it off and on

**Comment by BerkieBb** [comment]
Yeah I understood that

**Comment by BerkieBb** [comment]
I've committed https://github.com/BerkieBb/bMenu/commit/2661b72fd22391b80ba77f13fbcd416fee2cf62a to ensure the menu position does set on the main menu, all other menus with perms are already re-registered each time the perms are checked, so those are good already. Now I need to finish the perms to make sure other menus do set the menu position too.

**Comment by BerkieBb** [comment]
https://github.com/BerkieBb/bMenu/commit/e2ca668d93121dae4aadf8e2edf11cf095c23f99
This should fix your Display Location error

**Comment by izmystic** [comment]
going to start working on this again, is there a todo list somewhere?

**Comment by BerkieBb** [comment]
Still in client/main.lua
